### PR TITLE
[NCL-7537] Upgrade PatternFly from v4 to v5

### PR DESCRIPTION
**Pull request by patrikk0123**

### CODE REVIEW

File: src/components/BuildsList/BuildsList.tsx [hunk 1]
```diff
@@ -181,7 +181,7 @@ export const BuildsList = ({ serviceContainerBuilds, columns = defaultColumns, c
               id={StorageKeys.isBuildsListCompactMode}
               label="Compact Mode"
               isChecked={isCompactMode}
-              onChange={(checked) => {
+              onChange={(_, checked) => {
```
Comment (jomrazek) [inline src/components/BuildsList/BuildsList.tsx:184]: Out of curiosity, are you saying by "_" that there is no event?

**Review by jomrazek — APPROVED** [review]
Looks ok to my knowledge

**Review by matedo1 — APPROVED** [review]
Looks good, I added a few more details to the jira comment

File: src/components/BuildsList/BuildsList.tsx (hunk 1 above)
Comment (patrikk0123) [reply src/components/BuildsList/BuildsList.tsx:184]: the event is there, `_` or string prepended by `_`, is a convention to name variables to denote that the value is not being used (linter if properly set complains the variable is declared but unused)

File: src/components/AdministrationPage/AdministrationPage.tsx [hunk 2]
```diff
@@ -172,6 +156,7 @@ export const AdministrationPage = () => {
                     autoComplete="off"
                     {...register<string>(pncStatusEntityAttributes.banner.id)}
                   />
+                  <FormInputHelperText variant="error">{getFieldErrors(pncStatusEntityAttributes.banner.id)}</FormInputHelperText>
```
[the remark this answers is not in the document]
Comment (patrikk0123) [reply src/components/AdministrationPage/AdministrationPage.tsx:159]: PF `FormHelperText` component does not require it nor has any prop for it (has no `isHidden`)
if the children of that component are empty (including an empty string) the helper is not displayed

I included the `isHidden` prop on `FormInputHelperText` when the helper string is not strictly empty but has to be disabled by other logic

**Pull request merged**